template/source/PlayFab/Source/PlayFabCpp/Private/Core/PlayFabCppBaseModel.cpp.ejs

```diff
@@ -95,7 +95,8 @@ FDateTime PlayFab::readDatetime(const TSharedPtr<FJsonValue>& value)
     FString DateString = value->AsString();
     if (!FDateTime::ParseIso8601(*DateString, DateTimeOut))
     {
-        UE_LOG(LogPlayFabCpp, Error, TEXT("readDatetime - Unable to import FDateTime from Iso8601 String"));
+        UE_LOG(LogPlayFabCpp, Error, TEXT("readDatetime - Unable to import FDateTime from Iso8601 String. If you see a date time of all 0's, investigate the value being passed in and handle possible mis-parsing of ParseIso8601."));
+        DateTimeOut = FDateTime(0, 0, 0, 0, 0, 0, 0);
```

Comment:
>FDateTime(0, 0, 0, 0, 0, 0, 0) [](http://example.com/codeflow?start=22&length=30)

is this different than a default constructed FDateTime? what does the caller code look like to detect this situation?

IMO (outside of a breaking change that changes the API signature) the best thing to do is return a default constructed value and relnote it.  In that case title code would look something like:

auto dateTime = PlayFab::readDatetime(...);
if (dateTime == FDateTime{}) {
    // Handle error here
}

Reply:
I have set the default to be FDateTime::MinValue() which should result to January 1 0001 00:00:00 Log mentions two ways to check this, so callers should look something like

auto dateTime = PlayFab::readDateTime(...);
if(dateTime == FDateTime::MinValue()) { 
 // fill in expected dateTime handling here
 }